Gemfile.lock

```diff
@@ -118,7 +118,7 @@ GEM
 PLATFORMS
   ruby
   x86_64-linux
-  x86_64-linux-musl
```

Comment:
Was it a mistake to commit this Gemfile change? If it was I'd make a new commit with whats on master instead so we don't end up with such changes that may go untested.

Reply:
Accident, I'll fix.